src/CLRunner.ts

```diff
@@ -140,6 +143,27 @@ export class CLRunner {
         return newRunner;
     }
 
+    private static validiatedModelOptions(modelOptions?: Partial<ModelOptions.CLModelOptions>): ModelOptions.CLModelOptions {
+        const sessionTimout = (!modelOptions
+            || !modelOptions.sessionTimout
+            || typeof modelOptions.sessionTimout !== 'number')
+            ? ModelOptions.DEFAULT_MAX_SESSION_LENGTH
+            : modelOptions.sessionTimout
+
+        const repromptThreshold = (!modelOptions
+            || !modelOptions.repromptThreshold
+            || typeof modelOptions.repromptThreshold !== 'number')
+            || +modelOptions.repromptThreshold < 0
```

Comment:
I think finding name for `repromptThreshold` that has the word 'score' in it to let users knows it's related to the `scoreVariance`.

Reply:
Can't remember if we added retry count or not, But I initially thought this was an integer specifying max retries threshold.  Think `repromptScoreVarianceMinimum` is a bit long but more clear about what it's a threshold for.